reverse-proxy.md

```diff
@@ -41,14 +41,13 @@ Add this as a new Apache site config:
 <VirtualHost *:443>
     ServerName <your-nc-domain>
 
-    # Reverse proxy
-    RewriteEngine On
-    ProxyPreserveHost On
+    # Reverse proxy based on https://httpd.apache.org/docs/current/mod/mod_proxy_wstunnel.html
+    RewriteEngine on
+    ProxyPass / http://localhost:11000/
     RewriteCond %{HTTP:Upgrade} websocket [NC]
     RewriteCond %{HTTP:Connection} upgrade [NC]
-    RewriteRule ^/(.*) "ws://localhost:11000/$1" [P,L]
-    ProxyPass / http://localhost:11000/
-    ProxyPassReverse / http://localhost:11000/
+    RewriteRule ^/?(.*) "ws://localhost:11000/$1" [P,L]
+    ProxyPreserveHost On
```

Comment:
This seems to work for ws connections in general. Only Collabora still fails and I really do not understand why.

Reply:
It still fails with 

`WebSocket connection to 'wss://cloud.lindnercloud.com/cool/https%3A%2F%2Fcloud.lindnercloud.com%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F4_oc7k52n1zdg3%3Faccess_token%3DGFt6n7UK5HGxa83XTVo0g4fegz6kvzAB%26access_token_ttl%3D0%26permission%3Dedit/ws?WOPISrc=https%3A%2F%2Fcloud.lindnercloud.com%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F4_oc7k52n1zdg3&compat=/ws' failed:`

Reply:
Maybe try off here of this option

Reply:
Also ProxyRequests off